simtools/layout/telescope_position.py

```diff
@@ -293,7 +293,7 @@ def _convert(self, crs_from, crs_to, xx, yy):
             raise
         if xx is None or yy is None:
             return np.nan, np.nan
-        _to_x, _to_y = transformer.transform(xx=xx, yy=yy)
+        _to_x, _to_y = transformer.transform(xx, yy)
```

Comment:
This one I don't actually agree with.

Reply:
You are right, my mistake. Misunderstood the pylint messages.